api/export_pr.go

```diff
@@ -96,6 +104,14 @@ func (pr *PullRequest) ExportData(fields []string) map[string]interface{} {
 			data[f] = pr.Labels.Nodes
 		case "projectCards":
 			data[f] = pr.ProjectCards.Nodes
+		case "projectItems":
```

Comment:
We usually have these field names map closely to the underlying GraphQL schema, but that's not a strict contract. In this case, I wonder if the `projectCards` vs. `projectItems` distinction does any service to the user. Offering both without any documentation clarifying that one is Projects Classic and another is Projects Next could be confusing.

Could we just merge ProjectItems and expose it under `projectCards`?

Reply:
Agreed that the distinction is confusing. We definitely could merge the two. I think there are two downsides. 

The first is that new projects don't use "cards", so it might be confusing for users who are familiar with only projects V2. 

Second, is that the data we want to return for each is slightly different. They have the title in common but that is about it. What if in the future we want to expand on what data gets returned for `projectItems`? I don't envision `projectCards` getting expanded as projects classic usage dwindles.

I think if we didn't have to worry about backwards compatibility I would suggest making a new `projects` name that would encompass both, while removing `projectCards`. Since we do, I would perhaps leave it as `projectCards` and `projectItems` while adding some help documentation describing the difference.

Reply:
Good point about "cards" not being a term that new-style Projects use. Now that you reasoned about this, I see value in `projectCards` and `projectItems` being distinct.